Replace progress bar in offline examples

Getting reports that app crashes when the user opens the offline examples due to the progress bar. Lets use something more compatible with older devices. 

Android Version: Android 4.4
phone: Galaxy Note

```
java.lang.RuntimeException: Unable to start activity ComponentInfo{com.mapbox.mapboxandroiddemo/com.mapbox.mapboxandroiddemo.examples.SimpleOfflineMapActivity}: android.view.InflateException: Binary XML file line #20: Error inflating class android.widget.ProgressBar
	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2413)
	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2471)
	at android.app.ActivityThread.access$900(ActivityThread.java:175)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1308)
	at android.os.Handler.dispatchMessage(Handler.java:102)
	at android.os.Looper.loop(Looper.java:146)
	at android.app.ActivityThread.main(ActivityThread.java:5603)
	at java.lang.reflect.Method.invokeNative(Native Method)
	at java.lang.reflect.Method.invoke(Method.java:515)
	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:1283)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1099)
	at dalvik.system.NativeStart.main(Native Method)
Caused by: android.view.InflateException: Binary XML file line #20: Error inflating class android.widget.ProgressBar
	at android.view.LayoutInflater.createView(LayoutInflater.java:626)
	at com.android.internal.policy.impl.PhoneLayoutInflater.onCreateView(PhoneLayoutInflater.java:56)
	at android.view.LayoutInflater.onCreateView(LayoutInflater.java:675)
	at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:700)
	at android.view.LayoutInflater.rInflate(LayoutInflater.java:761)
	at android.view.LayoutInflater.inflate(LayoutInflater.java:498)
	at android.view.LayoutInflater.inflate(LayoutInflater.java:398)
	at android.view.LayoutInflater.inflate(LayoutInflater.java:354)
	at android.support.v7.app.AppCompatDelegateImplV7.setContentView(AppCompatDelegateImplV7.java:280)
	at android.support.v7.app.AppCompatActivity.setContentView(AppCompatActivity.java:140)
	at com.mapbox.mapboxandroiddemo.examples.SimpleOfflineMapActivity.onCreate(SimpleOfflineMapActivity.java:38)
	at android.app.Activity.performCreate(Activity.java:5458)
	at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1093)
	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2377)
	... 11 more
Caused by: java.lang.reflect.InvocationTargetException
	at java.lang.reflect.Constructor.constructNative(Native Method)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at android.view.LayoutInflater.createView(LayoutInflater.java:600)
	... 24 more
Caused by: android.content.res.Resources$NotFoundException: File res/drawable/panel_picture_frame_background.xml from drawable resource ID #0x1080629
	at android.content.res.Resources.loadDrawable(Resources.java:3434)
	at android.content.res.TypedArray.getDrawable(TypedArray.java:602)
	at android.view.View.<init>(View.java:3714)
	at android.widget.ProgressBar.<init>(ProgressBar.java:269)
	at android.widget.ProgressBar.<init>(ProgressBar.java:262)
	at android.widget.ProgressBar.<init>(ProgressBar.java:258)
	... 27 more
Caused by: java.lang.NullPointerException
	at android.graphics.drawable.DrawableContainer$DrawableContainerState.addChild(DrawableContainer.java:584)
	at android.graphics.drawable.StateListDrawable$StateListState.addStateSet(StateListDrawable.java:291)
	at android.graphics.drawable.StateListDrawable.inflate(StateListDrawable.java:189)
	at android.graphics.drawable.Drawable.createFromXmlInner(Drawable.java:990)
	at android.graphics.drawable.Drawable.createFromXml(Drawable.java:930)
	at android.content.res.Resources.loadDrawable(Resources.java:3430)
	... 32 more
```

--- a/MapboxAndroidDemo/src/main/res/layout/activity_offline_manager.xml
+++ b/MapboxAndroidDemo/src/main/res/layout/activity_offline_manager.xml
@@ -25,7 +25,7 @@
         android:layout_centerHorizontal="true"
         android:paddingLeft="25dp"
         android:paddingRight="25dp"
-        style="@android:style/Widget.Material.ProgressBar.Horizontal" />
+        style="?android:attr/progressBarStyleHorizontal" />
 
     <LinearLayout
         android:layout_width="match_parent"

--- a/MapboxAndroidDemo/src/main/res/layout/activity_offline_simple.xml
+++ b/MapboxAndroidDemo/src/main/res/layout/activity_offline_simple.xml
@@ -19,7 +19,7 @@
 
     <ProgressBar
         android:id="@+id/progress_bar"
-        style="@android:style/Widget.Material.ProgressBar.Horizontal"
+        style="?android:attr/progressBarStyleHorizontal"
         android:layout_width="match_parent"
         android:layout_height="25dp"
         android:layout_centerHorizontal="true"

